sideboard/server.py

```diff
@@ -3,61 +3,33 @@
 import sys
 
 import six
-#import ldap
```

Comment:
We've had a bunch of commented-out and dead code that's literally never been used in the history of Sideboard but which was sitting around because it got copied in from an older project.  I finally deleted it.  Sideboard plugins are welcome to use LDAP (one of them where I work totally does) but I don't think it makes sense to bundle it into the project, and we now support custom authenticators so it's super-easy to do this.